vnext/Desktop/module.g.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "winrt/base.h"
+void* winrt_make_facebook_react_MemoryTracker();
+void* winrt_make_facebook_react_NativeLogEventSource();
+void* winrt_make_facebook_react_NativeTraceEventSource();
+
+bool __stdcall winrt_can_unload_now() noexcept
+{
+    if (winrt::get_module_lock())
+    {
+        return false;
+    }
+
+    winrt::clear_factory_cache();
+    return true;
+}
+
+void* __stdcall winrt_get_activation_factory([[maybe_unused]] std::wstring_view const& name)
+{
+    auto requal = [](std::wstring_view const& left, std::wstring_view const& right) noexcept
```

Comment:
>requal [](start = 9, length = 6)

(nit) why not just spell out "AreEqual"?

Reply:
C++/WinRT code.